packages/fe-fpm-writer/templates/page/1.84/manifest.json

```diff
@@ -0,0 +1,34 @@
+{
+    "sap.ui5": {
+        "dependencies": {
+            "minUI5Version": "1.84.0"
+        },
```

Comment:
@tobiasqueck can we remove also this hardcoded version?

Reply:
6e537f9